<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QWidget" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="m_pReturnButton">
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_CreateButton">
          <property name="text">
           <string>创建文件夹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_DeleteButton">
          <property name="text">
           <string>删除文件夹</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_RenameButton">
          <property name="text">
           <string>重命名文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_FlushDirButton">
          <property name="text">
           <string>刷新文件夹</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="m_UploadButton">
          <property name="text">
           <string>上传文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_DeleteFileButton">
          <property name="text">
           <string>删除文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_DownloadButton">
          <property name="text">
           <string>下载文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ShareButton">
          <property name="text">
           <string>分享文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
